From 8dafd2c36ae8ed71577e4ff3c7c7ade18f670e37 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Wed, 16 Dec 2015 08:19:46 +0100
Subject: [PATCH] Add missing QMap and QDate includes

BirthdayPage has class members of these types, so we need to make sure
their definition is available.
---
 Settings/BirthdayPage.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Settings/BirthdayPage.h b/Settings/BirthdayPage.h
index 2fb5325..a738622 100644
--- a/Settings/BirthdayPage.h
+++ b/Settings/BirthdayPage.h
@@ -21,6 +21,8 @@
 
 // Qt includes
 #include <QWidget>
+#include <QMap>
+#include <QDate>
 
 // Qt classes
 class QLineEdit;
-- 
2.6.4

